test/active_support_test_case_helper.rb

```diff
@@ -39,6 +40,10 @@ def skip_if_plugin_asked_to
     end
   end
 
+  def fetch_proxy_features
+    ProxyAPI::Features.any_instance.stubs(:features => Feature.name_map.keys)
```

Comment:
This adds a new SQL query to every single test, as it's called regardless of whether the stubbing's needed or not. What's it used for? Can the stubbing be added to the tests that need it?